plasma_framework/contracts/src/exits/payment/PaymentExitGame.sol

```diff
@@ -21,9 +21,14 @@ contract PaymentExitGame is IExitProcessor, PaymentStandardExitRouter {
     {
     }
 
-    function processExit(uint192 _exitId) external {
-        if (ExitId.isStandardExit(_exitId)) {
-            PaymentStandardExitRouter.processStandardExit(_exitId);
+    /**
+     * @dev for ERC20, thus each erc contract address represent a token value
```

Comment:
updated, please take a look. 

as for `ercContract`, I am thinking to rename `token` to `ercContract`(or anything more suitable)  in `PlasmaFramework` layer (see #273). However, as in this `PaymentExitGame`, it will always be representing `token` so probably it's fine to just use the `token` as function args? how do you think?